bots/tests-scan

```diff
@@ -386,14 +417,15 @@ def scan_for_pull_tasks(api, policy, opts, repo):
         sys.stderr.write("tests-scan: No /dev/kvm access, not running tests here\n")
         return []
 
-    results = cockpit_tasks(api, not opts.dry, policy, repo, opts.pull_number)
+    results = cockpit_tasks(api, not opts.dry, policy, repo, opts.pull_number, opts.sha, opts.repo)
 
     if opts.human_readable:
         func = lambda x: tests_human(*x)
         results.sort(reverse=True, key=lambda x: str(x))
-    else:
-        func = lambda x: tests_invoke(*x, options=opts)
-    return list(map(func, results))
+        return list(map(func, results))
+    with distributed_queue(opts.amqp, opts.queue) as queue:
+        func = lambda x: queue_test(*x, queue=queue, options=opts)
+        return list(map(func, results))
```

Comment:
I think this needs to get along with not specifying an amqp server/queue, and then work as it does now. This is both for an easier transition to queued, as well as developing locally and running tests-scan with `-d`.